README.adoc

```diff
@@ -47,6 +47,15 @@ To install glide, go to the
 link:https://github.com/Masterminds/glide/releases[release page] and download
 the newest binary for your operating system.
 
+Glide expects `GOPATH` environment variable to be set.  It also requires
+`PATH` environment updated with `$GOPATH/bin`. e.g.,
```

Comment:
Look at the sentence below:

>  Unpack the archive that you've downloaded and place the `glide` executable
  somewhere so that it is in your `PATH`.		  somewhere so that it is in your `PATH`.

It already says everything and really we should not spend time with helping people how to install glide. https://glide.sh/ has excellent documentation for that.